packages/atomic/cypress/integration/facets-v1/numeric-facet/numeric-facet-assertions.ts

```diff
@@ -129,3 +129,122 @@ function getEvenRangeValue(value: string) {
   const end = Number(value.split(' to ')[1].replaceAll(',', ''));
   return end - start;
 }
+
+export function assertCurrencyFormat() {
+  it('should display correct currency format CA$', () => {
+    NumericFacetSelectors.valueLabel().as('facetAllValuesLabel');
+    cy.getTextOfAllElements('@facetAllValuesLabel').then((originalValues) => {
+      originalValues.forEach((e: string) => {
+        expect(e.split(' to ')[0]).contains('CA$');
+        expect(e.split(' to ')[1]).contains('CA$');
+        expect(e.split(' to ')[0].split('$')[0]).eq('CA');
+      });
+    });
+  });
+}
+
+export function assertUnitFormatKgLong() {
+  it('should display correct unit format "kilograms"', () => {
+    NumericFacetSelectors.valueLabel().as('facetAllValuesLabel');
+    cy.getTextOfAllElements('@facetAllValuesLabel').then((originalValues) => {
+      originalValues.forEach((e: string) => {
+        expect(e.split(' to ')[0]).contains('kilograms');
+        expect(e.split(' to ')[1]).contains('kilograms');
+        expect(e.split(' to ')[0].split(' ')[1]).eq('kilograms');
+      });
+    });
+  });
+}
+
+export function assertUnitFormatHourNarrow() {
+  it('should display correct unit format "kg"', () => {
+    NumericFacetSelectors.valueLabel().as('facetAllValuesLabel');
+    cy.getTextOfAllElements('@facetAllValuesLabel').then((originalValues) => {
+      originalValues.forEach((e: string) => {
+        expect(e.split(' to ')[0]).contains('kg');
+        expect(e.split(' to ')[1]).contains('kg');
+        expect(e.split(' to ')[0].slice(-2)).eq('kg');
+      });
+    });
+  });
+}
```

Comment:
Shouldn't this be `hr` instead of `kg`?

Reply:
nice catch, thanks, the title of the test should be 'kilogram' :)